src/auto-evo/simulation/food_source/RandomEncounterFoodSource.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace AutoEvo
+{
+    /// <summary>
+    ///   A class for random-encounter based food sources.
+    /// </summary>
+    public abstract class RandomEncounterFoodSource : FoodSource
+    {
+        protected override float StorageScore(MicrobeSpecies species, Compound compound, Patch patch,
+            SimulationCache simulationCache)
+        {
+            // TODO compute value based on encouter chance ; this is a temporary solution for day/night cycle PR.
```

Comment:
```suggestion
            // TODO: compute value based on encouter chance; this is a temporary solution for day/night cycle PR.
```
Also I doubt this will end up being "temporary" because if this is not done properly now, I'll predict that in like 3 years this "temporary" code will still be used.